src/routes/Agreements/AgreementsView.js

```diff
@@ -55,9 +57,9 @@ class AgreementsView extends PureComponent<Props, StateProps> {
               align='center'
             />
           </h1>
-          <div className='items'>
+          <div>
             {CONDITIONS_LIST.map((key: string) => (
-              <div className='item' key={key}>
+              <div className={style.checkboxItem} key={key}>
```

Comment:
This name is used purely as example of camelCase coercion. It would be a lot more simple to call it just `item`.